src/actions/classifier.js

```diff
@@ -25,29 +25,106 @@ export function startNewClassification(workflowID) {
     }).then(() => {
       return dispatch(setNeedsTutorial())
     }).then(() => {
+      dispatch(setState('loadingText', 'Loading Subjects...'))
+      return dispatch(loadSubjects())
+    }).then(() => {
+      return dispatch(setSubjectsToDisplay())
+    }).then(() => {
+      //now we can create the first classification!!
+      let subject = getState().classifier.subject[workflowID]
+      let workflow = getState().classifier.workflow[workflowID]
+      return apiClient.type('classifications').create({
+        annotations: [],
+        metadata: {
+          workflow_version: workflow.version,
+          started_at: (new Date).toISOString(),
+          user_agent: `${Platform.OS} Mobile App`,
+          user_language: 'en', //TODO: Will be fixed subsequent PR
+          utc_offset: ((new Date).getTimezoneOffset() * 60).toString(),
+          subject_dimensions: []
+        },
+        links: {
+          project: workflow.links.project,
+          workflow: workflow.id,
+          subjects: [subject.id]
+        }
+      })
+    }).then((classification) => {
+      dispatch(setState(`classifier.classification.${workflowID}`, classification))
       dispatch(setState('classifier.isFetching', false))
-    }).catch(() => {
+    }).catch((e) => {
+      console.log('Error in catch! ', e)
```

Comment:
Is this necessary? I believe this would be the only `console.log` in the app, and I'm not sure where it would be visible.